chrome/content/zotero/xpcom/style.js

```diff
@@ -44,11 +44,19 @@ Zotero.Styles = new function() {
 	this.ns = {
 		"csl":"http://purl.org/net/xbiblio/csl"
 	};
-
-	// TEMP
-	// Until we get asynchronous style loading, load renamed styles at startup, since the
-	// synchronous call we were using breaks the first drag of the session (on OS X, at least)
+	
 	this.preinit = function () {
+		// Upgrade style locale prefs for 4.0.27
```

Comment:
It's really just a note so we know why there's an upgrade (because of changes in 4.0.27). The code is pretty clear that the upgrade happens independently of actual version checks. Feel free to change the comment if you want. It seems ok to me.